packages/core/test/client.test.ts

```diff
@@ -275,6 +275,30 @@ describe('@bugsnag/core/client', () => {
       client.notify(new Error('oh em eff gee'))
     })
 
+    it('includes duration in event.app', done => {
+      const start = +new Date()
+      const client = new Client({ apiKey: 'API_KEY_YEAH', enabledReleaseStages: ['staging'], releaseStage: 'staging' })
+
+      // Delay sending the notification by this many milliseconds to ensure
+      // 'duration' will always have a useful value
+      const delayMs = 10
+
+      client._setDelivery(client => ({
+        sendEvent: (payload) => {
+          // The maximum number of milliseconds 'duration' should be
+          const maximum = +new Date() - start
+
+          expect(payload.events[0].app.duration).toBeGreaterThanOrEqual(delayMs)
+          expect(payload.events[0].app.duration).toBeLessThanOrEqual(maximum)
+
+          done()
+        },
+        sendSession: () => {}
+      }))
+
+      setTimeout(() => client.notify(new Error('oh em eff gee')), delayMs)
```

Comment:
Not a big deal since it's only 10ms but an alternative to using an actual time delay is to consider using jest fake timers.

Reply:
This looks possible in Jest 26 with ['modern' timer mocks](https://jestjs.io/docs/en/jest-object#jestusefaketimersimplementation-modern--legacy) but we're on version 25 — updating might be out of scope of this PR